File: app/src/main/java/com/agarwal/arpit/androidhub/lifecycle/ActivityB.java
package com.agarwal.arpit.androidhub.lifecycle;

import android.os.Bundle;

import com.agarwal.arpit.androidhub.R;

import androidx.appcompat.app.AppCompatActivity;

public class ActivityB extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_b);
    }
}

File: app/src/main/java/com/agarwal/arpit/androidhub/lifecycle/ActivityB.kt
package com.agarwal.arpit.androidhub.lifecycle

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.agarwal.arpit.androidhub.R


class ActivityB : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_b)
    }


}
